cores/rp2040/SerialPIO.cpp

```diff
@@ -346,6 +351,14 @@ void SerialPIO::flush() {
     delay((1000 * (_txBits + 1)) / _baud);
 }
 
+void SerialPIO::setInverted(bool i){
+		_inverted = i;
+}
+
+bool SerialPIO::getInverted(){
```

Comment:
At some point in my life I decided I shouldn't write setters without matching getters. One hopes the compiler will strip it out if it's not used. In this case I was using it for a flip-state test command, where otherwise I have to duplicate the boolean state externally to know it. 
It would be even simpler to just scope the _inverted property public. 
But whatever meets your design goals, I'm happy to do.  I don't need that getter any more, will remove.